README.md

```diff
@@ -182,8 +182,13 @@ To create a `standard.x2` (or other non-default types) instance type, the `plan`
 managed_kafka.sh --create mykafka --plan standard.x2
 ```
 
-### Deploying to multiple clusters/zones
-In order to allow clients to deploy Kafka clusters to multiple clusters and/or zones, the following parameters must be customized:
+### Deploying to a single provider and region
+The default installation configurations will deploy kas-fleet-manager with a single cloud provider and region. The
+provider name and region may be configured in the environment using the `CLOUD_PROVIDER` and `REGION` variables. See
+the `kas-installer-defaults.env` for the default values.
```

Comment:
If `gcp` is configured as a cloud provider then gcp credentials should be provided too. See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1299 for details on how to do it.

Aside from that, `gcp` should be enabled as a supported cloud provider through the `SUPPORTED_CLOUD_PROVIDERS` env var

Reply:
Thanks for calling that out. I'm thinking we can add a link here to the KFM GCP configuration docs (and any others that may pertain to cloud-provider specific configs) and leave it as an exercise for the reader to determine how to use them. My concern with putting the specifics in this README is that it would be a repeat of the maintained docs that KFM has and would likely become out of date. w.d.y.t?

Reply:
We could do that, but the linked doc url might end up out of sync with the version of kfm used.

Reply:
I'm curious how they're used. I didn't need to use them when installing to a GCP cluster a few months ago.

I agree that the link can get out of date. We might be better off to generically reference the KFM docs for more information on how to configure cloud provider-specific options.

Regarding the first comment in this thread, the idea is that for a single-cluster deployment, the user can provide `CLOUD_PROVIDER=gcp` and `REGION=us-east1` alone to deploy as a GCP cluster. That avoids needing to set the more complex JSON configurations for simple cases.

Reply:
:+1:

Reply:
Added another paragraph to indicate that additional cloud provider configuration may be needed in some situations and that the KFM docs should be consulted.